### PR TITLE
docs: remove duplicate contribute

### PR DESCRIPTION
**Pull request by tabcat**
There are identical Contribute sections before and after the License section of the README. This PR removes the former with its ToC entry.

### CODE REVIEW

**Review by achingbrain — CHANGES_REQUESTED** [review]
Thanks for opening this, could you please remove the entry from the table of contents at the top of the file as well?

File: README.md [hunk 1]
```diff
@@ -20,7 +20,6 @@
   - [As a custom runtime](#as-a-custom-runtime)
 - [Releasing a new version](#releasing-a-new-version)
 - [Interop release process for when breaking changes are introduced](#interop-release-process-for-when-breaking-changes-are-introduced)
-- [Contribute](#contribute)
 - [License](#license)
 - [Contribute](#contribute-1)
```
Comment (achingbrain) [inline README.md:24]:
```suggestion
- [Contribute](#contribute)
```

**Review by achingbrain — APPROVED** [review]

**Comment by achingbrain** [comment]
Thanks!

**Comment by tabcat** [comment]
thanks for catching those parts i missed. next time ill be more thorough.

**Pull request merged**